x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/components/auto_follow_pattern_table/auto_follow_pattern_table.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiInMemoryTable,
+  EuiButton,
+} from '@elastic/eui';
+
+export class AutoFollowPatternTableUI extends PureComponent {
+  static propTypes = {
+    autoFollowPatterns: PropTypes.array,
+  }
+
+  state = {
+    selectedItems: [],
+  }
+
+  onSearch = ({ query }) => {
+    const { text } = query;
+    const normalizedSearchText = text.toLowerCase();
+    this.setState({
+      queryText: normalizedSearchText,
+    });
+  };
+
+  getFilteredPatterns = () => {
+    const { autoFollowPatterns } = this.props;
+    const { queryText } = this.state;
+
+    if(queryText) {
+      return autoFollowPatterns.filter(autoFollowPattern => {
+        const { __id__, remote_cluster } = autoFollowPattern; // eslint-disable-line camelcase
+
+        if (__id__.toLowerCase().toLowerCase().includes(queryText)) {
+          return true;
+        }
+
+        if (remote_cluster.toLowerCase().toLowerCase().includes(queryText)) {
+          return true;
+        }
+
+        return false;
+      });
+    }
+
+    return autoFollowPatterns.slice(0);
+  };
+
+  getTableColumns() {
+    const editField = () => {
+      console.log('Coming soon!');
+    };
+
+    const { intl } = this.props;
+
+    return [{
+      field: '__id__',
+      name: (
+        <FormattedMessage
+          id="xpack.cross_cluster_replication.autoFollowPatternList.table.nameColumnTitle"
```

Comment:
probably shouldn't mix snake and camel case in ids for i18n, should all be camel case I think.

Reply:
Ok will change that no prob. @cjcenizal we might want to align the remote cluster app too for the translation keys